pkg/util/image_prep_utils.go

```diff
@@ -79,29 +75,17 @@ func getImageFromTar(tarPath string) (string, error) {
 func CleanupImage(image Image) {
 	if image.FSPath != "" {
 		glog.Infof("Removing image filesystem directory %s from system", image.FSPath)
-		errMsg := remove(image.FSPath, true)
-		if errMsg != "" {
-			glog.Error(errMsg)
+		if err := remove(image.FSPath, true); err != nil {
+			glog.Error(err.Error())
```

Comment:
This can just be `glog.Error(err)`

Also, this won't exit or do anything besides print the error to stdout and the glog error file.   Is that what you mean to do here?

Reply:
not new code, but yeah this can fail without impacting the overall run. this should probably get knocked down to a warn though